agent/base

```diff
@@ -270,12 +270,11 @@ function resolve_benchmark_bin {
 }
 
 function interrupt() {
-    # On abnormal exit, stop the tool meister ...
-    pbench-tool-meister-stop
+    # On abnormal exit, stop the tool meister ane make sure the metadata log
+    # exists and is complete.
+    pbench-tool-meister-stop "int"
```

Comment:
Bug, we don't have the "group" variable to stop the tool meisters in the interrupt context, so we'll need to discover it properly, or make sure all users of `interrupt` use the same tool group variable name.

Reply:
Fixed.